cmd/agent/api/pb/api.proto

```diff
@@ -15,12 +15,18 @@ service Agent {
 }
 
 service AgentSecure {
-    rpc GetTags (TagRequest) returns (TagReply) {
+    rpc TaggerStreamEntities(StreamTagsRequest) returns (stream StreamTagsResponse) {
         option (google.api.http) = {
-            post: "/v1/grpc/tags"
+            post: "/v1/grpc/stream_tags"
```

Comment:
I would maybe put these in: `/v1/grpc/tagger/stream_tags` and `/v1/grpc/tagger/fetch_entity`

Reply:
Also, you might want to add some docs (or comments) somewhere describing what would be the expected JSON format of the messages when the grpc-gateway is used. I know this won't apply much to practical use, but it might be helpful for troubleshooting. We're more likely to send a customer a short command-line curl snippet rather than ask them to install `grpcurl`, and so forth.

Reply:
Finally, with regards to the fact the tagger endpoint is going to be secured. Will this always be the case, or do you anticipate scenarios were the tagger might have to be accessed without authentication? 

There are currently two alternatives for scenarios were access to the gRPC (or legacy API) is required from remote locations:
1. shared secret
2. unauthenticated API access (still encrypted though)

If it could be either/or, we might have to think a little bit about how to expose them.

Reply:
Thanks for the review @truthbk! I applied the changes to the gateway paths as you suggested, and added documentation.

Regarding authentication, I don't think we'll ever want the tagger endpoints to ever be split between authenticated and non-authenticated. It's possible that we might want to remove auth at some point, but if that ever happens we'll move the whole thing (as in, move all the endpoints from `AgentSecure` to `Agent`), to avoid having to do complicated routing and RBAC.

For remote access, we expect a similar mechanism to that of the DCA, through a shared secret mounted at a known location for the clients, which as I understand is what's implemented for the `serverSecure` server.